sdk/servicebus/Azure.Messaging.ServiceBus/Directory.Build.props

```diff
@@ -3,4 +3,16 @@
     Add any shared properties you want for the projects under this package directory that need to be set before the auto imported Directory.Build.props
   -->
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory).., Directory.Build.props))\Directory.Build.props" />
+  
+  <PropertyGroup>
+    <!--
+    These analyzers are blocked by https://github.com/Azure/azure-sdk-tools/issues/127
+     -->
+    <NoWarn>
+      $(NoWarn);

```

Comment:
This allows us to not match the ClientOption type names exactly with the Client type.
This is useful for SenderClientOptions.